packages/editor/src/client/atoms/editorTemplate.tsx

```diff
@@ -22,7 +22,7 @@ export function EditorTemplate({children, navigationChildren}: EditorTemplatePro
           display: 'flex',
           position: 'sticky',
           top: 0,
-          zIndex: 3,
+          zIndex: 10,
```

Comment:
Why have you changed this one? :eyes:

Reply:
This one is to make the plus buttons appear behind the metadata and the save buttons when scrolling.

Reply:
You mean disappear?

Reply:
yes as it currently passes on top of the buttons.
![image](https://user-images.githubusercontent.com/27450825/123247788-880cb480-d4e7-11eb-8bfe-39573ca89bc1.png)